src/main/java/com/solvd/web/GameZonePage.java

```diff
@@ -0,0 +1,27 @@
+package com.solvd.web;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class GameZonePage extends AbstractPage {
```

Comment:
there is no need to specify which page this is. This should be a universal page that we will use for any category from the catalog